cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,187 @@
+:_mod-docs-content-type: ASSEMBLY
+[id=“cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing”]
+= Tutorial: ROSA with HCP private offer acceptance and sharing
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing
+
+toc::[]
+
+This guide describes how to accept a private offer for {hcp-title-first} and how to ensure that all team members can benefit from the private offer for the clusters they provision.
+
+{hcp-title} costs are composed of the AWS infrastructure costs, such as the EC2 instances that are running the needed workloads, and the {hcp-title} service costs AWS infrastructure costs are charged to the AWS account where the infrastructure is deployed.
```

Comment:
I think it needs a period after "service costs" and before "AWS infrastructure"

Reply:
I think it was a little confusing my first read to figure out what the cost components were. I would recommend moving the example to the second sentence to make it clear on the first sentence. Here is my recommendation:

"ROSA with HCP costs are composed of the AWS infrastructure costs and the ROSA with HCP service costs. AWS infrastructure costs, such as the EC2 instances that are running the needed workloads, are charged to the AWS account where the infrastructure is deployed. ROSA service costs are charged to the AWS account specified as the “AWS billing account” when deploying a cluster.

The cost components can be billed to different AWS accounts. Detailed description of how the ROSA service cost and AWS infrastructure costs are calculated can be found on the [Red Hat OpenShift Service on AWS Pricing page](https://aws.amazon.com/rosa/pricing/)."